yt/frontends/gizmo/fields.py

```diff
@@ -32,25 +32,20 @@
                   "Mg", "Si", "S", "Ca", "Fe"]
 
 class GizmoFieldInfo(GadgetFieldInfo):
+    # The known fields list is according to the GIZMO User Guide. See
+    # http://www.tapir.caltech.edu/~phopkins/Site/GIZMO_files/gizmo_documentation.html#snaps-reading
     known_particle_fields = (
-        ("Mass", ("code_mass", ["particle_mass"], None)),
```

Comment:
So we're removing these previously-defined fields because they're not in the base-level gizmo file?

Reply:
I'm doing this mostly to make the list cleaner and more maintainable because the list now maps from the GIZMO User Manual directly. I have checked the first commit of the GIZMO code and believe there are no historical GIZMO snapshots with these variant field names. It would be great if I could get further confirmation from you.